testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -1113,7 +1152,19 @@ impl RelayerThread {
                     info!("In initial block download, will not submit VRF registration");
                     return true;
                 }
-                self.rotate_vrf_and_register(&last_burn_block);
+                let mut saved_key_opt = None;
+                if let Some(path) = self.config.miner.activated_vrf_key_path.as_ref() {
+                    saved_key_opt =
+                        Self::load_saved_vrf_key(&path, &self.keychain.get_nakamoto_pkh());
+                }
+                if let Some(saved_key) = saved_key_opt {
+                    debug!("Relayer: resuming VRF key");
+                    self.globals.resume_leader_key(saved_key);
+                } else {
+                    debug!("Relayer: directive Register VRF key");
+                    self.rotate_vrf_and_register(&last_burn_block);
+                    debug!("Relayer: directive Registered VRF key");
```

Comment:
I think this should just be one `debug!` line.

Reply:
I copied that log sandwich from the neon node. Shall I remove one of the logs there too while I'm at it?

Reply:
Done in 869179c2688fcd18d77a17fab7edef1e9e50b2e5. I also added another log change unrelated to this comment, 9369da9f811b3147d9a50700dd1b964f92395b13, which I noticed when debugging (it was printing an array of bytes instead of the hex value).